terraform/monitoring/config/alert.test.yml

```diff
@@ -49,3 +51,38 @@ tests:
               exp_labels:
                 environment: production
                 severity: high
+    - interval: 1m
+      # Series data.
+      input_series:
+          - series: 'response_time_bucket{app="teaching-vacancies-production",status_range="2xx",le="1" }'
+            values: '0 0 0 0 0 0 10 20 30 40 50 50 50 50 50 50'
+          - series: 'response_time_bucket{app="teaching-vacancies-production",status_range="2xx",le="5" }'
+            values: '0 0 0 0 0 0 10 20 30 40 50 60 70 80 90 100'
+          - series: 'response_time_bucket{app="teaching-vacancies-production",status_range="2xx",le="10" }'
+            values: '0 0 0 0 0 0 10 20 30 40 50 60 70 80 90 100'
+          - series: 'response_time_bucket{app="teaching-vacancies-production", status_range="2xx",le="+Inf"}'
+            values: '0 0 0 0 0 0 10 20 30 40 50 60 70 80 90 100'
+
+      # Name of the test group
+      name: Test slow_response alert
```

Comment:
The alert name is "ProdSlowRequest" so you probably want to write "Test ProdSlowRequest alert"